romancal/tweakreg/tweakreg_step.py

```diff
@@ -74,60 +72,13 @@ class TweakRegStep(RomanStep):
         abs_fitgeometry = option('shift', 'rshift', 'rscale', 'general', default='rshift')
         abs_nclip = integer(min=0, default=3) # Number of clipping iterations in fit when performing absolute astrometry
         abs_sigma = float(min=0.0, default=3.0) # Clipping limit in sigma units when performing absolute astrometry
-        output_use_model = boolean(default=True)  # When saving use `DataModel.meta.filename`
     """  # noqa: E501
 
     reference_file_types = []
 
     def process(self, input):
-        use_custom_catalogs = self.use_custom_catalogs
```

Comment:
Are we removing the option to pass user catalogs?

Reply:
No, that was a mistake.